### ISSUE TITLE
[BUG] Spurious "aria-label is not valid for the element <div> with implicit ARIA role generic" error

### ISSUE THREAD

**Issue by francispf94** [IBMa/equal-access#2179]
**Project**
accessibility-checker for Node

**Operating system**

MacOS: 15.2 (24C101)

Chrome: 132.0.6834.84 (Official Build) (arm64)

IBM Equal Access Accessibility Checker: 3.1.79

**Description**

I get a spurious error when running from Node:

`The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic"`

The accessibility violation occurs in the [Layer](https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/Layer/index.tsx) component from React Carbon when the aria-label prop is passed to it.

It works when run from Chrome.

**Versions**

> franciscofernandes@Franciscos-MacBook-Pro lineage-ui % npm list accessibility-checker
lineage-ui@ /Users/franciscofernandes/Documents/lineage-ui
└── accessibility-checker@3.1.79 overridden
franciscofernandes@Franciscos-MacBook-Pro lineage-ui % node --version
v20.18.1
franciscofernandes@Franciscos-MacBook-Pro lineage-ui % npm list jsdom
lineage-ui@ /Users/franciscofernandes/Documents/lineage-ui
└─┬ jest-environment-jsdom@29.7.0
  └── jsdom@25.0.1 overridden

**Comment by francispf94** [comment]
No longer an issue, can be closed.
